ceph/rados/core_workflows.py

```diff
@@ -5367,3 +5367,26 @@ def lookup_log_message(
             msg_logline = f"The log line {search_string} not found on - {daemon_type} : {daemon_id} log"
             log.info(msg_logline)
             return False
+
+    def remove_log_file_content(self, nodes_list):
+        """
+        The method is used to remove the log file contents in the cluster
+        Args:
+            nodes_list: Cluster nodes list that remove the file content
+
+        Returns: True -> Successful execution of command
+                 False -> Failure of command
+
+        """
+        cluster_fsid = self.run_ceph_command(cmd="ceph fsid")["fsid"]
+        cmd_truncate = rf"cd /var/log/ceph/{cluster_fsid}/ && find . -type f -exec truncate -s 0 {{}} \;"
```

Comment:
Please exclude compressed files
Add ability to truncate logs of specific daemons, accept input from user